### ISSUE TITLE
CSS_Ref broken smartlinks

### ISSUE THREAD

**Issue by caugner** [mdn/yari#8578]
While testing out https://github.com/mdn/yari/pull/8571, Sentry collected the following errors regarding broken smartlinks in the `CSS_Ref` macro:

 `In CSS_Ref the smartLink to ... is broken!`

- [ ] `/en-US/docs/Web/CSS/:host()` (redirects to /en-US/docs/Web/CSS/:host_function)
- [ ] `/en-US/docs/Web/CSS/:host-context()` (does not exist)
- [ ] `/en-US/docs/Web/CSS/@font-face/font-feature-settings` (does not exist)
- [ ] `/en-US/docs/Web/CSS/@font-face/font-variant` (redirects to /en-US/docs/Web/CSS/@font-face)
- [ ] `/en-US/docs/Web/CSS/@historical-forms` (does not exist)
- [ ] `/en-US/docs/Web/CSS/@page/bleed` (does not exist)
- [ ] `/en-US/docs/Web/CSS/an-plus-b` (does not exist)
- [ ] `/en-US/docs/Web/CSS/block-overflow` (does not exist)
- [ ] `/en-US/docs/Web/CSS/caret` (does not exist)
- [ ] `/en-US/docs/Web/CSS/caret-shape` (does not exist)
- [ ] `/en-US/docs/Web/CSS/conic-gradient` (redirects to /en-US/docs/Web/CSS/gradient/conic-gradient)
- [ ] `/en-US/docs/Web/CSS/hwb` (does not exist)
- [ ] `/en-US/docs/Web/CSS/image-set` (redirects to /en-US/docs/Web/CSS/image/image-set)
- [ ] `/en-US/docs/Web/CSS/input-security` (does not exist)
- [ ] `/en-US/docs/Web/CSS/single-transition-timing-function#cubic-bezier()` (redirects to /en-US/docs/Web/CSS/easing-function)

**Comment by caugner** [comment]
FYI @wbamberg

**Comment by wbamberg** [comment]
Yeah, this is because CSS_Ref uses mdn/data to get the list of items to include, and the things in mdn/data don't match the actual pages we have.

It would probably be better for CSS_Ref to use what's actually under /CSS, along with page types, to build https://developer.mozilla.org/en-US/docs/Web/CSS/Reference#index. Assuming we think https://developer.mozilla.org/en-US/docs/Web/CSS/Reference#index is useful enough to be worth it, which I doubt.
